index.html

```diff
@@ -4,9 +4,184 @@
+            </svg>
+          </span>
+          <h3 class="sub-progran__name">Workshop</h3>
+          <p class="sub-progran__description">
+            This is free workshop is a step by step guide to help land a remote
+            job.
+          </p>
+        </div>
+        <div class="card card--primar">
+          <span>
+            <svg class="icon icon-primary">
+              <use xlink:href="./images/sprite.svg#forum"></use>
+            </svg>
+          </span>
+          <h3 class="sub-progran__name">Forum</h3>
+          <p class="sub-progran__description">
+            Have the time to share your thoughts and opinions with experts for
+            each topic.
+          </p>
+        </div>
+        <div class="card card--primar">
+          <span>
+            <svg class="icon icon-primary">
+              <use xlink:href="./images/sprite.svg#increasing-users"></use>
+            </svg>
+          </span>
+          <h3 class="sub-progran__name">Our clients</h3>
+          <p class="sub-progran__description">
+            Our previous clients are building careers at some of the top
+            companies in tech.
+          </p>
+        </div>
+        <div class="card card--primar">
+          <span>
+            <svg class="icon icon-primary">
+              <use xlink:href="./images/sprite.svg#conference-education"></use>
+            </svg>
+          </span>
+          <h3 class="sub-progran__name">Networing</h3>
+          <p class="sub-progran__description">
+            Make connections and build relationships with our experts.
+          </p>
+        </div>
+      </div>
+      <a href="#" class="btn btn-text">SEE THE WHOLE PROGRAM</a>
+      <button class="btn btn--primary btn-block">Join the course</button>
+    </section>
+
+    <!-- featured-section -->
+    <section class="featured-section"></section>
+
+    <!-- partner-section -->
+    <section class="partner-section">
+      <div class="partner-section__header">
+        <h2 class="partner-section__title">featured Speakers</h2>
+        <div class="red-line"></div>
+      </div>
+      <div class="partner-section__body">
+        <ul class="partner-section__list">
+          <li class="partner-section__list-item">
+            <i class="fab fa-google" aria-hidden="true">oogle</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-airbnb" aria-hidden="true"> airbnb</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-microsoft" aria-hidden="true"> Microsoft</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-apple" aria-hidden="true"> Apple</i>
+          </li>
+          <li class="partner-section__list-item">
+            <i class="fab fa-free-code-camp" aria-hidden="true"></i>
+            freeCodeCamp
+          </li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] [REQUIRED] Please ensure that you stick closely to the color of the provided template. The entire partner sections look too faded on your design. You applied `#3e3e3e` as the section's background color, kindly reduce its intensity so that the logos will be well seen. Implement this in order to have a better UI. Please implement for the mobile section as well.
### Your Design
![8](https://user-images.githubusercontent.com/58692590/159014564-4c83eee2-b96a-4342-b3a6-b8454cee40f0.png)
### Provided Template
![88](https://user-images.githubusercontent.com/58692590/159014618-205c6c9f-6931-4b7c-8420-d0a10128c030.png)

Reply:
Thank you, i did that